docs/resources/config.md

```diff
@@ -0,0 +1,35 @@
+---
+page_title: "doppler_config Resource - terraform-provider-doppler"
+subcategory: ""
+description: |-
+	Manages a Doppler config.
+---
+
+# doppler_config (Resource, Pre-Release)
+
+**This resource is in pre-release and is not intended to be used in production.**
+
+Manages a Doppler config.
```

Comment:
Would it make sense to use the imperative here- "Manage a Doppler config"

Reply:
Hashicorp's own resources use the present simple ("manages") but others, such as [Splunk](https://registry.terraform.io/providers/splunk/splunk/latest/docs/resources/inputs_tcp_raw) use the imperative.

I hadn't given it much thought but the imperative feels more natural to me here. I'll make the change 👍